media-sound/musescore/musescore-9999.ebuild

```diff
@@ -47,22 +50,29 @@ DEPEND="
 	dev-qt/qtquickcontrols2:5
 	>=dev-qt/qtsingleapplication-2.6.1_p20171024[X]
 	dev-qt/qtsvg:5
+	dev-qt/qtx11extras:5
 	dev-qt/qtxml:5
 	dev-qt/qtxmlpatterns:5
+	>=media-libs/alsa-lib-1.0.0
```

Comment:
there's no need to specify min version when or lowest version is greater, atm 1.2.8

Reply:
It's okay to specify it if it's in somewhat recent memory (it's needed to get upgrade order right, especially on older systems), but for something as old as <=1.0.0, no point indeed.

Reply:
That was just copied over from 4.0.2 (which in turn was just a copy of the dependency spec of 3.6.2). I'll adjust that.